lib/src/widget/canvas.dart

```diff
@@ -123,8 +123,11 @@ class _DiagramEditorCanvasState extends State<DiagramEditorCanvas>
       children: [
         ...showBackgroundWidgets(),
         ...showOtherWithComponentDataUnder(canvasModel),
+        if(widget.policy.displayLinksInFrontOfComponents())
```

Comment:
This would not work because both conditions are equal. Please add a `!` to the second one.

```dart
        if (widget.policy.showLinksOnTopOfComponents) ...showComponents(canvasModel),
        ...showLinks(canvasModel),
        if (!widget.policy.showLinksOnTopOfComponents) ...showComponents(canvasModel),
```